docs/se/guides/40_webclient.adoc

```diff
@@ -0,0 +1,420 @@
+----
+package io.helidon.examples.quickstart.se;
+
+public class ClientExample {
+
+    public static void main(String[] args) {
+
+    }
+}
+----
+
+Add the following code to create a WebClient instance. The builder approach allow us to create the WebClient with
+specific settings. It improves the readability and simplicity of the code.
+
+[source,java]
+.Add WebClient instance to the main method:
+----
+import io.helidon.media.jsonp.JsonpSupport;
+import io.helidon.webclient.WebClient;
+
+...
+
+WebClient webClient = WebClient.builder()
+                .baseUri("http://localhost:8080")   // <1>
+                .addMediaSupport(JsonpSupport.create())     // <2>
+                .build();
+----
+<1> The base URI of the outbound requests.
+<2> Register a support for Jsonp.
+
+By default, Helidon quickstart run on the host localhost and port 8080. If you changed these values in configuration
+file, make sure to modify the base URI as well.
+Once build, the WebClient can be used to send a GET request to the greeting application.
+
+[source,java]
+.Send a GET request to the target endpoint:
+----
+webClient.get()     // <1>
+    .path("/greet")     // <2>
+    .request(String.class)  // <3>
+    .peek(System.out::println)  // <4>
+    .await();   // <5>
+----
+<1> Create a Http GET request.
+<2> Target endpoint path.
+<3> Execute the request and return Single with response entity handled as a String.
+<4> Print the response in the console.
+<5> Wait for server response because of reactive approach.
+
+
+
+The path method joins `/greet` to the WebClient base Uri. The target Uri for this request becomes
+`http://localhost:8080/greet` where the response should be a greeting message. Received response entity will be
+automatically handled as a String. If no specific type is put into the method request(),
+WebClientResponse is returned by default. This WebClientResponse object contains response code, headers and non-handled
+entity.
+
+==== Run the application
+
+[source,bash]
+.Build the quickstart:
+----
+mvn package
+----
+
+This command will create helidon-quickstart-se.jar in the target folder.
+
+[source,bash]
+.Run the greeting application first:
+----
+java -cp target/helidon-quickstart-se.jar io.helidon.examples.quickstart.se.Main
+----
+
+Open a new command prompt or terminal and run the ClientExample class you just created.
+
+[source,bash]
+.Run the greeting application first:
+----
+java -cp target/helidon-quickstart-se.jar io.helidon.examples.quickstart.se.ClientExample
+----
+
+[source,bash]
+.Output:
+----
+{"message":"Hello World!"}
+----
+
+When the ClientExample finishes its execution, you can stop the Main class by pressing `ctrl+c`.
+
+==== Discover other WebClient functionality
+
+In practice, String is not the most useful return type, since it usually needs some more handling.In this case it could
+be more interesting to return an object such as JsonObject. Earlier, Json support was added to the webclient,
```

Comment:
In the previous step JSON support was added to the WebClient so that it could be used instead of String.